main.nf

```diff
@@ -428,10 +420,21 @@ process fastqc {
     script:
     prefix = reads.baseName
     """
-    echo ${prefix}
+#    echo `which gunzip`
 
     fastqc $reads
-		extract_fastqc_stats.sh --srr=${prefix} > ${prefix}_stats_fastqc.txt
+		#extract_fastqc_stats.sh --srr=${prefix} > ${prefix}_stats_fastqc.txt
+#    GC=\$(gunzip -c "\$(find . -name *_fastqc.zip)" "${prefix}"_fastqc/fastqc_data.txt \
```

Comment:
All of the outcommented stuff can basically be deleted. Git has your back, in case you need it at some point anymore.

Reply:
Fixed.